src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/ApplicationEntity.kt

```diff
@@ -515,8 +518,8 @@ interface ApplicationSummary {
   fun getId(): UUID
   fun getCrn(): String
   fun getCreatedByUserId(): UUID
-  fun getCreatedAt(): Timestamp
-  fun getSubmittedAt(): Timestamp?
+  fun getCreatedAt(): Instant
+  fun getSubmittedAt(): Instant?
```

Comment:
Added a link to the PR description however there isn't a lot of content in there.

I'm sure you know this in any case @davidatkinsuk but mainly for my reference..  From what I can see upgrading Spring results in a complaint that `Timestamp` isn't zoned and therefore we are making them zoned by converting to `Instant`.  `Instant` is zoned since it is UTC (by default) since it represents a point in time reference from unix epoch (which itself is UTC).

I have checked:

```
import java.sql.Timestamp
import java.time.Instant
import java.time.ZoneOffset

fun main() {
    val milli = System.currentTimeMillis()
    val timestampDate = Timestamp(milli).toLocalDateTime().toLocalDate()
    val instantDate = Instant.ofEpochMilli(milli).atZone(ZoneOffset.UTC).toLocalDate()
    assert(timestampDate == instantDate)
}
```

Reply:
I'm happy with this. I've been doing some debugging to compare what's happening internal with timezones in new/old hibernate and it seems consistent - https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/5017108537/Use+of+DateTime+and+Timezone+API+Kotlin+JPA+Postgres